packaging/installer/netdata-updater.sh

```diff
@@ -439,7 +442,10 @@ self_update() {
 
 parse_version() {
   r="${1}"
-  if echo "${r}" | grep -q '^v.*'; then
+  if [ "${r}" = "latest" ]; then
+    printf "99999999999999"
```

Comment:
Is there any special motive to use this `magic number`?

Reply:
It’s guaranteed to be larger than any other version number that the function might produce. I should probably add a comment to that effect though.

Reply:
Please, add a comment to simplify our understanding.